lib/internal/cluster/worker.js

```diff
@@ -20,14 +19,16 @@ function Worker(options) {
 
   this.exitedAfterDisconnect = undefined;
 
+  // This property has been deprecated and is being removed entirely
+  // in Node.js 9.0.0
```

Comment:
I am not sure yet that the correct action would be to remove the property entirely in 9.0.
Apart from this comment, this change looks fine.